apollo-http-cache/src/test/kotlin/com/apollographql/apollo3/cache/http/internal/CachingHttpInterceptorTest.kt

```diff
@@ -122,6 +122,31 @@ class CachingHttpInterceptorTest {
       }
     }
   }
+
+  @Test
+  fun cacheInParallel() {
```

Comment:
Nice test 👍
Out of curiosity, would it be possible to write it using `launch {}` and `Dispatchers.IO` instead of `Executors`? This way, the test would be KMP-ready?

Reply:
My first try but without `Dispatchers.IO` which didn't repro the issue - not sure why (isn't the default dispatcher also multithreaded 🤔), but with `Dispatchers.IO` it works! 👍